packages/client/src/store/userStore.ts

```diff
@@ -159,6 +161,19 @@ export const useUserStore = create<IUserStore>()(
           addToast((error as IEError).response.data.reason);
         }
       },
+      // Oauth
+      oauthLogin: async (code: IOauthSignInRequest) => {
+        try {
+          await ApiClient.signInWithYandex(code);
+          const data = await ApiClient.getUserInfo();
+          set(() => ({
+            user: {
+              ...data,
+              isAuth: true,
+            },
+          }));
+        } catch {}
```

Comment:
Было бы хорошо хотя бы кидать тостер с ошибкой, чем совсем ничего не делать. Иначе смысл улавливания ошибки теряется
```suggestion
        } catch {
          addToast("Failed to login OAuth")
        }
```

Reply:
добавил